docker-build.md

```diff
@@ -168,23 +201,24 @@ To build and tag your image, open a Terminal (Mac/Linux) or Command
 Prompt (Windows) and navigate to the folder that contains your
 Dockerfile:
 
-```
+```shell script
```

Comment:
what does this do?

Reply:
hook for syntax highlighting, if it ever gets enabled

Reply:
I don't want syntax highlighting here though, because it's a shell *command* not a *script*. I think?

Reply:
Hmmm... I think it will do "the right thing" and highlight the `cd`... but I'm not sure. I'll remove it for now.